src/librustdoc/clean/mod.rs

```diff
@@ -955,6 +956,43 @@ fn clean_ty_generics<'tcx>(
     }
 }
 
+fn clean_ty_typealias_inner_type<'tcx>(
```

Comment:
`typealias` without an `_` is not that great, what about just

```suggestion
fn clean_ty_alias_inner_type<'tcx>(
```
```